arangod/Aql/Ast.cpp

```diff
@@ -1114,9 +1114,12 @@ AstNode* Ast::createNodeValueString(std::string const &value) {
     return const_cast<AstNode*>(&EmptyStringNode);
   }
 
+  auto const length = value.length();
```

Comment:
you can replace the **entire** method body with `return `createNodeValueString(value.data(), value.size());`

Reply:
Copy&paste ftw